### ISSUE TITLE
[RFC]: add .rwignore support or new group in redwood.toml for excluding folders from rw scanning 

### ISSUE THREAD

**Issue by bnn1** [redwoodjs/redwood#7761]
### Summary

Follow up from #7690
Add a file `.rwignore` (or whatever) that will "tell" RW scanner to skip certain folders. 

Other option would be to add a new group in `redwood.toml`, say named "cli" with one variable "scanIgnoreDirs" which accepts an array of strings - paths user want to exclude from scanning
```toml
[cli]
  scanIgnoreDirs = ["db"]
```

### Motivation

When in the RW project directory there is a folder with restricted permission, RW fails to generate anything because it can't scan folders with restricted permission

Such a folder can be created by running docker containers with their volume mapped to somewhere in the RW project directory.

We need a way to tell RW to not scan certain folders to avoid crashing.

### Detailed proposal

To address the issue of the RW tool failing when encountering a folder it cannot read, we propose adding a new group in the redwood.toml configuration file called "cli" or add support for `.rwignore` file. Within this group, we can add a variable called `scanIgnoreDirs`, which accepts an array of strings representing the paths that the user wants to exclude from scanning. This will allow users to specify which directories the RW tool should ignore while scanning.

Here is an example of how the scanIgnoreDirs variable can be used:

```toml
[cli]
scanIgnoreDirs = ["db", "node_modules"]
```

In this example, the db and node_modules directories will be excluded from scanning. Users can add as many directories as they want to this array.

By implementing this feature, we can provide more control to the user in terms of which directories should be ignored while scanning. This will make the RW tool more robust and less prone to failing due to permission issues.

Adding the scanIgnoreDirs variable to the redwood.toml file will allow users to exclude specific directories from being scanned by the RW tool. This will make the tool more robust and less prone to failing due to permission issues. The implementation of this feature should be straightforward and can be completed in a timely manner.

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by Tobbe** [comment]
More support for this feature has been raised in 
https://github.com/redwoodjs/redwood/issues/8752
and
https://github.com/redwoodjs/redwood/issues/5403#issuecomment-1337695776

**Comment by Tobbe** [comment]
We don't have the bandwidth to start working on this feature right now, but we do like the idea.

@hacknaked or @bnn1 if either of you (or anyone else) is interested in working on this that'd be awesome. Feel free to get started whenever you like 🙂 We'll have someone from the team to help guide you if you need, and to help with reviewing the code etc

**Comment by bnn1** [comment]
@Tobbe I found the culprit, fixing it now.

**Comment by bnn1** [comment]
The issue has been fixed in #9079
